packages/calcite-components/src/components/tile-group/tile-group.stories.ts

```diff
@@ -0,0 +1,964 @@
+import { select, boolean } from "@storybook/addon-knobs";
+import { html } from "../../../support/formatting";
+import readme from "./readme.md";
+import { storyFilters } from "../../../.storybook/helpers";
+
+export default {
+  title: "Components/Tiles/Tile Group",
+  parameters: {
+    notes: readme,
+    chromatic: { delay: 10000 },
+  },
+  ...storyFilters(),
+};
+
+export const playground = (): string => html`
+  <calcite-tile-group
+    dir="${select("dir", ["ltr", "rtl"], "ltr", "Tile Group")}"
+    ${boolean("disabled", false)}
+    layout="${select("layout", ["horizontal", "vertical"], "horizontal", "Tile Group")}"
+    scale="${select("scale", ["s", "m", "l"], "m")}"
+  >
+    <calcite-tile
+      heading="Tile heading lorem ipsum"
+      description="Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collab on thinking to further the overall."
+      icon="layers"
+    ></calcite-tile>
+    <calcite-tile
+      heading="Tile heading lorem ipsum"
+      description="Leverage agile frameworks to provide a robust synopsis for high level overviews."
+      icon="layers"
+    ></calcite-tile>
+    <calcite-tile
+      heading="Tile heading lorem ipsum"
+      description="Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collab on thinking to further the overall."
+      icon="layers"
+    ></calcite-tile>
+    <calcite-tile
+      heading="Tile heading lorem ipsum"
+      description="Iterative approaches to corporate strategy foster collab."
+      icon="layers"
+    ></calcite-tile>
+  </calcite-tile-group>
+`;
+
+export const allVariants_NoTest = (): string => html`
```

Comment:
Can you create a follow-up issue to include this in screenshot tests once we've addressed the `timeout` issue?

Reply:
https://github.com/Esri/calcite-design-system/issues/8829